### DISCUSSION TITLE
Future of node.js naming convention

### DISCUSSION THREAD

**Discussion by Phlaxith** [nodejs/nodejs.org#7255] [General]
Hello there ! 

I was wondering why the LTS node.js 22 name is "Jod". The previous versions codenames were inspired by materials. 💎
What is the meaning of Jod ? 🤔

**Comment by avivkeller** [comment]
Hi! This is an issue for nodejs/Release. In general, they are elements on the periodic table. Jod is German for Iodine

**Reply by Phlaxith** [reply]
Thank you for the reply ! That's true phonetically is makes sense now ! 😁

**Comment by UlisesGascon** [comment]
You can find a lot of info here https://github.com/nodejs/Release/blob/main/CODENAMES.md, we use chemical elements for the naming :+1: 

> In early periodic tables, iodine was often given the symbol J, for Jod, its name in German; in German texts, J is still frequently used in place of I
> [wikipedia](https://en.wikipedia.org/wiki/Iodine#:~:text=In%20early%20periodic%20tables%2C%20iodine,used%20in%20place%20of%20I.)

"Q" and "W" are under discussion: https://github.com/nodejs/Release/issues/304

**Reply by ocmstone** [reply]
The only one I can find that starts with "Q" is again German - Quecksilber (Quicksilver, or mercury).  Sounds cool enough to me!!

**Reply by ocmstone** [reply]
And once again, thanks to our German brethren, "W" is for Wolfram - or tungsten.
